pkg/gds/emails/client.go

```diff
@@ -26,6 +26,14 @@ func New(conf config.EmailConfig) (m *EmailManager, err error) {
 		client: sendgrid.NewSendClient(conf.SendGridAPIKey),
 	}
 
+	// Warn if email configuration isn't complete and will produce partial emails.
+	if conf.VerifyContactBaseURL == "" || conf.AdminReviewBaseURL == "" {
+		log.Warn().
+			Bool("missing_verify_contact_base_url", conf.VerifyContactBaseURL == "").
+			Bool("missing_admin_review_base_url", conf.AdminReviewBaseURL == "").
+			Msg("partial email configuration, some emails may not include links")
+	}
```

Comment:
When the email client is initialized, that's when we're issuing warnings about partial configs now. Note that the configs are still optional, but the admins should be alerted.